tests/robot-tests/tests/libs/admin/manage-content-common.robot

```diff
@@ -23,6 +23,121 @@ user creates new content section
+    user checks headed table body row cell contains    Number of open settings    1    22,900
+    user checks headed table body row cell contains    Proportion of settings open    1    1%
+
+    user enters text into element    id:dataBlockDetailsForm-name    ${datablock_name}
+    user enters text into element    id:dataBlockDetailsForm-heading    ${datablock_title}
+    user enters text into element    id:dataBlockDetailsForm-source    Dates source
+
+    user clicks button    Save data block
+
+    user waits until h2 is visible    Edit data block
+    user waits until page contains button    Delete this data block
+
+user creates key stats data block for dates csv
+    [Arguments]
+    ...    ${subject_name}
+    ...    ${datablock_name}
+    ...    ${datablock_title}
+    ...    ${indicator_name}
+    ...    ${expected_indicator_value}
+
+    user clicks link    Data blocks
+    user waits until h2 is visible    Data blocks
+
+    user clicks link    Create data block
+    user waits until h2 is visible    Create data block
+
+    user waits until table tool wizard step is available    1    Choose a subject
+    user waits until page contains    ${subject_name}
+    user clicks radio    ${subject_name}
+    user clicks element    id:publicationSubjectForm-submit
+
+    user waits until table tool wizard step is available    2    Choose locations
+    user opens details dropdown    National
+    user checks location checkbox is checked    England
+
+    user clicks element    id:locationFiltersForm-submit
+
+    user waits until table tool wizard step is available    3    Choose time period
+    user chooses select option    id:timePeriodForm-start    2020 Week 13
+    user chooses select option    id:timePeriodForm-end    2020 Week 13
+    user clicks element    id:timePeriodForm-submit
+
+    user waits until table tool wizard step is available    4    Choose your filters
+    user clicks subheaded indicator checkbox    Open settings    ${indicator_name}
+    user checks subheaded indicator checkbox is checked    Open settings    ${indicator_name}
+
+    user opens details dropdown    Date
+    user clicks category checkbox    Date    23/03/2020
+    user checks category checkbox is checked    Date    23/03/2020
+
+    user clicks element    id:filtersForm-submit
+    user waits until results table appears    %{WAIT_LONG}
+
+    user checks table column heading contains    1    1    2020 Week 13
+    user checks headed table body row cell contains    ${indicator_name}    1    ${expected_indicator_value}
+
+    user enters text into element    id:dataBlockDetailsForm-name    ${datablock_name}
+    user enters text into element    id:dataBlockDetailsForm-heading    ${datablock_title}
+    user enters text into element    id:dataBlockDetailsForm-source    Dates source
+
+    user clicks button    Save data block
+
+    user waits until h2 is visible    Edit data block
+    user waits until page contains button    Delete this data block
```

Comment:
I'd be tempted to append `%{WAIT_MEDIUM}` to all of the waits in this PR to avoid transient failures in CI

Reply:
Thanks @luke-h1 good idea.  I've added to strategic points in the new code.  Swapped a lot of "waits for" lines for "checks that" instead where I would not expect it to need to wait any further as well, if you just have a look at the changes in the final commit on this PR?